src/app/Services/checkIn.service.ts

```diff
@@ -3,47 +3,56 @@ import { HttpClient, HttpParams } from '@angular/common/http';
 import { environment } from 'src/environments/environment';
 import { User } from '../Models/user'
 import { ok } from 'assert';
+import { NGXLogger } from 'ngx-logger';
 
 @Injectable({
   providedIn: 'root'
 })
 export class CheckInService {
 
-  constructor(private httpClient: HttpClient) { }
+  constructor(private httpClient: HttpClient, private logger: NGXLogger) { }
 
   checkIn(pin: number) {
     if (environment.production == true) {
+      this.logger.info("Attempt to Check In, with pin " + pin.toString() + " (POST in Production)");
```

Comment:
The message displayed, in my opinion is not appropriate. In no matter you should show the pin that has been typed. Even if it's a mistake

Reply:
We need to discuss it

Reply:
yes

Reply:
changed logger implementation